apps/studio/prisma/seed.ts

```diff
@@ -3,19 +3,13 @@
  *
  * @link https://www.prisma.io/docs/guides/database/seed-database
  */
-import { PrismaClient } from '@prisma/client'
-
-const prisma = new PrismaClient()
+import { db } from '~/utils/database'
 
 async function main() {
   // Nothing
 }
 
-main()
-  .catch((e) => {
-    console.error(e)
-    process.exit(1)
-  })
-  .finally(async () => {
-    await prisma.$disconnect()
-  })
+main().catch((e) => {
+  console.error(e)
+  process.exit(1)
+})
```

Comment:
shall we .finally(disconnect)? if not failed instances might have dangling connections liao

Reply:
oh yes, will add

Reply:
done